core/controller.go

```diff
@@ -204,7 +218,7 @@ func (c *Controller) sendStart(w http.ResponseWriter, success bool, stream *stre
 		return
 	}
 	logrus.Infof("%s started processing | StartHandler", stream.OriginalURI)
-	b, _ := json.Marshal(SummariseDTO{URI: stream.Path, Running: true, ID: stream.ID})
+	b, _ := json.Marshal(SummariseDTO{URI: stream.Path, Running: true, ID: stream.ID, Alias: stream.ID})
```

Comment:
Is it intended to use the ID as alias? I'm pretty sure here we should check the alias map if the stream has any.

Reply:
Yes, I set the preference to promote the alias over the uuid for a consistent Uri for clients. The check is downstream, but here we use the id as the alias. I could change the start payload to include an optional alias value as well if you prefer.